Makefile

```diff
@@ -162,7 +162,10 @@ install-hub-agent-helm:
     --set namespace=fleet-system \
     --set enableWebhook=true \
     --set webhookServiceName=fleetwebhook \
-    --set webhookClientConnectionType=service
+    --set webhookClientConnectionType=service \
+    --set enableV1Alpha1APIs=true \
+	--set enableV1Beta1APIs=false \
```

Comment:
why add this? This is not v1alpha1 test

Reply:
This is for the v1alpha1. For v1beta1 is in setup.sh